doc/tutorials/klondike/app/.metadata

```diff
@@ -0,0 +1,42 @@
+# This file tracks properties of this Flutter project.
```

Comment:
This file should not be added since this is not an app that is being released, but just an example.

Reply:
Flutter-generated file. We need to eradicate commit https://github.com/flame-engine/flame/commit/18f71bece67fad79b71b9185dc18101f81f0df27 which unintentionally added and commited it.

Reply:
The offending commit https://github.com/flame-engine/flame/commit/18f71bece67fad79b71b9185dc18101f81f0df27 has been reverted.